src/Agent.Worker/CodeCoverage/CodeCoverageCommands.cs

```diff
@@ -20,6 +20,8 @@ public sealed class CodeCoverageCommandExtension : AgentService, IWorkerCommandE
         private string _codeCoverageTool;
         private string _reportDirectory;
 
+        public string SupportedHostType => "build";
```

Comment:
@omeshp if publish test result supported in Build/RM, why code coverage only supported in build, i thought TestResult and CodeCoverage has kind of same back end, do they?